### ISSUE TITLE
Feat: download from huggingface

### ISSUE THREAD

**Issue by sammcj** [sammcj/gollama#13]
Would tie in nicely with a modelfile templates, but could just pop up vim with a default template pre-populated with the basics.

**Comment by remon-nashid** [comment]
Would love to download HF models using gollama as the ollama registry is a bit restrictive. I see this issue is marked as complete but not sure how to access that feature. Any clues?

**Comment by sammcj** [comment]
Hey I just closed it off as thought maybe I skip looking into the idea, I'll consider it again but need to check overlap with other tools.

**Comment by remon-nashid** [comment]
That's great to hear @sammcj , it would be yet another limitation of ollama (access to a wider selection of ggufs) that gollama tackles. From my point of view, it's even handier than the bi-directional linking with LM Studio as the later feels like toe stepping at times, while with ollama, you're just providing a feature that wasn't there. Just my two cents.

**Comment by YugandharrPatil** [comment]
i completely agree @sammcj. i mean, how about we maintain a registry that we could continuously update, that gollama could pull from, which would contain code for automatic installation and configuration of "popular" models from huggingface. i mean the current state of gollama is okay; just a fancy wrapper around ollama CLI but this feature would be sick. especially for people who are not very well versed with python, pytorch, diffusers which is always a pain to setup.

**Comment by sammcj** [comment]
> i mean the current state of gollama is okay; just a fancy wrapper around ollama CLI

Why don't you contribute to make it more than that then?

**Comment by YugandharrPatil** [comment]
would love to if you're up for this idea. did u understand what I meant? will go through and understand the codebase soon!

**Comment by YugandharrPatil** [comment]
ohh and the "just" a fancy ollama wrapper part was constructive criticism, suggesting a new feature in the later part. in case you wrote that to provoke me, I don't really care man, it's your project.

**Comment by sammcj** [comment]
Not sure what's so constructive about it but ok.

If you wanted to have a go a contributing the feature you want I'd be happy to review it.

**Comment by sammcj** [comment]
I've been having a think about this feature and have decided not to implement it for now given Ollama's support for creating models from HF and the (painful) state of Ollama templates.
